interface/src/Application.cpp

```
@@ -2161,12 +2163,28 @@ void Application::initializeGL() {
         _isGLInitialized = true;
     }
 
+    // Build a shared canvas / context for the Chromium processes
     _glWidget->makeCurrent();
-    _chromiumShareContext = new OffscreenGLCanvas();
-    _chromiumShareContext->setObjectName("ChromiumShareContext");
-    _chromiumShareContext->create(_glWidget->qglContext());
-    _chromiumShareContext->makeCurrent();
-    qt_gl_set_global_share_context(_chromiumShareContext->getContext());
+    if (!nsightActive()) {
```

what happens when nsight is active? do web pages just silently fail? maybe we should include a qWarning() or something to indicate why.

---

added a warning and a comment to the code.